### PR TITLE
ADD an adapter to support Mariadb

### PR DESCRIPTION
**Pull request by hbarral**
Hello! I'm not a Vim expert, although I'd like to contribute. It's not much, it's just a copy of what's already out there, I don't deserve any credit for that.

This `pull request` is intended to support Mariadb based on recent executable name changes.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -13,6 +13,7 @@ take on [dbext.vim][], improving on it on the following ways:
   - jq
   - MongoDB
   - MySQL
+  - Mariadb
```
Comment (tpope) [inline README.markdown:16]: The correct spelling is "MariaDB".

**Review by tpope — CHANGES_REQUESTED** [review]
Needs to be added to the help file. Other than that this looks good to go.

File: README.markdown (hunk 1 above)
Comment (hbarral) [reply README.markdown]: It's already fixed, sir. Thank you!.

**Review by boydbloemsma — APPROVED** [review]

**Comment by boydbloemsma** [comment]
Can this be merged?

**Comment by Askath** [comment]
Would really love for this feature to be available soon

**Comment by raphaelladinig** [comment]
would be really nice if this would be merged soon. Tested it on my system and it work 100%.

**Comment by Askath** [comment]
Any info on this?

**Pull request merged**